### ISSUE TITLE
js购物车动画效果

### ISSUE THREAD

**Issue by zhangzhaoaaa** [zhangzhaoaaa/blog#40]
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Title</title>
	<script src="http://apps.bdimg.com/libs/jquery/2.1.4/jquery.js"></script>
	<script src="jquery.easing.js"></script>
	<style>
		div.panel{
			width: 400px;
			height: 500px;
			border: 1px solid #000;
			float: left;
		}
	</style>
</head>
<body>
<div class="panel">
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飞飞
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飘飘
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫爬爬
	</div>
</div>
<div class="panel target">
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飞飞
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飘飘
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫爬爬
	</div>
</div>
<div class="panel">
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飞飞
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫飘飘
	</div>
	<div class="item">
		<img class="photo" src="http://placekitten.com/150" width="100" height="100" />
		小猫爬爬
	</div>
</div>
<script>
	$('div.item').on('click', function(){
		var $orgImg = $('img', this);
		var $targetDiv = $('div.target');
	var $fxImg = $orgImg.clone().css({
	'position': 'absolute',
	'z-index': 10000,
	'width': $orgImg.width(),
	'height': $orgImg.height(),
	'border-radius': '50%'
	}).css($orgImg.offset()).appendTo('body');
	$fxImg
	 .animate({
		left: [$targetDiv.offset().left + $targetDiv.width() - $fxImg.width(), 'linear'],
		top: [$targetDiv.offset().top + $targetDiv.height() - $fxImg.height(), 'easeInBack'],
		width:[$orgImg.width()-50, 'linear'],
		height: [$orgImg.height()-50, 'linear']
	}, 600)
	.fadeOut(200, function () {
		$fxImg.detach();
	});
	});
</script>
</body>
</html>
